products/BellHybrid/apps/include/Application.hpp

```diff
@@ -29,6 +30,6 @@ namespace app
 
         virtual void onKeyPressed();
         virtual void onStart();
-        virtual void onStop();
+        bool idleTimerActiveFlag = true;
```

Comment:
is this change connected with this task?

Reply:
no, it sneaked in accidentally. Removed